test/integration_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/blevesearch/bleve"
 
 	// we must explicitly include any functionality we plan on testing
-	_ "github.com/blevesearch/bleve/analysis/analyzers/keyword_analyzer"
+	//_ "github.com/blevesearch/bleve/analysis/analyzers/keyword_analyzer"
```

Comment:
quickie double-check -- wondering if this was commented out line was on purpose?